src/ServiceControl/Infrastructure/RavenDB/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -28,6 +22,15 @@ public static Task<TimerJobExecutionResult> RunCleanup(int deletionBatchSize, Do
             logger.Debug("Trying to find expired EventLogItem documents to delete (with threshold {0})", threshold.ToString(Default.DateTimeFormatsToWrite, CultureInfo.InvariantCulture));
             EventLogItemsCleaner.Clean(deletionBatchSize, database, threshold, token);
 
+            if (settings.AuditRetentionPeriod.HasValue)
+            {
+                threshold = SystemTime.UtcNow.Add(-settings.AuditRetentionPeriod.Value);
```

Comment:
I know it was there before but what is the purpose of using RavenDB SystemTime vs DateTime.UtcNow?

Reply:
Same as above?